drivers/sensor/qdec_nrfx/qdec_nrfx.c

```diff
@@ -74,14 +74,16 @@ static int qdec_nrfx_sample_fetch(const struct device *dev,
 	return 0;
 }
 
+#define QDEC_STEPS DT_INST_PROP(0, steps)
```

Comment:
we should be using driver's config, it isn't great to put defines in between driver code.

Reply:
Sure, but that requires a bigger rewrite of the driver, let's leave that for another day :)

Reply:
it should be moved to the top, then, for consistency.